code/game/mecha/mecha_defense.dm

```diff
@@ -117,7 +117,13 @@
 	if(prob(deflect_chance))
 		severity++
 		log_append_to_last("Armor saved, changing severity to [severity].")
-	. = ..()
```

Comment:
To preserve current behavior of severity 1 explosions destroying the mech instead of deleting it into nothingness.